packages/checkbox/src/react/index.tsx

```diff
@@ -89,60 +122,43 @@ const Checkbox = React.forwardRef((props, forwardedRef) => {
       <input
         readOnly
         ref={ref}
-        tabIndex="-1"
+        tabIndex={-1}
         type="checkbox"
         value={props.value}
-        {...styles.input(themeName, props)}
-        {...filterReactProps(props, { tagName: 'input' })}
+        {...styles.input()}
+        {...omit<CheckboxProps, ['label', 'onCheck', 'error', 'indeterminate']>(
+          props,
+          ['label', 'onCheck', 'error', 'indeterminate']
+        )}
```

Comment:
👍 

there is probably a way to make the `omit` function figure these generic args out without the added code but i'm not that good at ts

Reply:
That would be awesome.  This was repetitive and didn't feel great.